salt/kubelet/kubelet.jinja

```diff
@@ -32,5 +44,6 @@ KUBELET_ARGS="\
     --cloud-provider=aws \
 {% endif -%}
     {{ pillar['components']['kubelet']['args'] }} \
+    --node-labels={{ node_labels|join(',') }} \
```

Comment:
What if `node_labels = []`? Can we provide `--node-labels=` to the kubelet args?